terraform/modules/emr/iam.tf

```diff
@@ -573,3 +590,71 @@ data "aws_iam_policy_document" "elastic_map_reduce_for_auto_scaling_role" {
     resources = ["*"] // Required by AutoScaling-CheckPermissions 
   }
 }
+
+data "aws_iam_policy_document" "group_hive_data_access_documents" {
+  for_each = var.security_configuration_groups
+
+  policy_id = "HiveData${replace(regex("((Non_)?PII)", each.key)[0], "_", "")}"
+
+  statement {
+    sid = "HiveDataS3${replace(regex("((Non_)?PII)", each.key)[0], "_", "")}"
+
+    actions = [
+      "s3:DeleteObject",
+      "s3:DeleteObjectVersion",
+      "s3:GetObject",
+      "s3:ListBucket",
+      "s3:PutObject",
+      "s3:ReplicateObject",
+      "s3:RestoreObject",
+    ]
+
+    resources = [
+      "${aws_s3_bucket.hive_data.arn}/${each.key}",
+      "${aws_s3_bucket.hive_data.arn}/${each.key}/*",
+    ]
+  }
+
+  statement {
+    sid = "HiveDataS3"
+    actions = [
+      "s3:GetBucketPublicAccessBlock",
+      "s3:ListBucket",
+      "s3:GetBucketVersioning",
+      "s3:ListMultipartUploadParts",
+      "s3:GetObject",
```

Comment:
Doesnt this let me read the data from another groups database? ie. non PII user able to read tables in the PII database?